src/components/ServiceDetails/InstallAndRunService/Go/index.js

```diff
@@ -1,10 +1,7 @@
 import React from "react";
-import { withStyles } from "@material-ui/styles";
-
-import { useStyles } from "./styles";
 
 const Go = ({ classes }) => {
```

Comment:
Do we need this component ?

Reply:
removed https://github.com/singnet/snet-dapp/pull/402/commits/10ec6a680f051b94b760f9716cd07759c93b4876